README.md

```diff
@@ -21,15 +21,27 @@ We're working on [tutorials](https://napari.org/tutorials/), but you can also qu
 
 ## installation
 
+### which distribution to install
+If you want to contribute back to napari codebase, you need to install from source code, see [from source](#from-source)
+
+If you do not want to use napari as python code and only use it as GUI app, the bundled app is the easiest way to install, 
+this is also the only method that does not require python knowledge to work with napari, see [from bundled app](https://napari.org/tutorials/fundamentals/installation#installing-as-a-bundled-app)
+
+If you are using napari while using python to programmatically interact with the app, you can either install via pip or source code, 
+however we recommend that you use conda to help managing the virtual environment, otherwise you may see compilation issues that are specific to machines.
```

Comment:
```suggestion
If you are using napari from Python to programmatically interact with the app, you can install via pip, conda-forge, or from source.
We recommend that you use conda to help manage the virtual environment. Otherwise you may see compilation issues that are specific to your particular machine, which is difficult for us to debug.
```